src/main/kotlin/no/nav/medlemskap/regler/v2/Hovedregler.kt

```diff
@@ -0,0 +1,63 @@
+package no.nav.medlemskap.regler.v2
+
+import no.nav.medlemskap.regler.common.Personfakta
+import no.nav.medlemskap.regler.common.Resultattype
+import no.nav.medlemskap.regler.v2.arbeidsforhold.RegelgruppeForArbeidsforhold
+import no.nav.medlemskap.regler.v2.common.Spørsmål
+import no.nav.medlemskap.regler.v2.common.Svar
+import no.nav.medlemskap.regler.v2.grunnforordningen.RegelgruppeForGrunnforordningen
+import no.nav.medlemskap.regler.v2.registrerte_opplysninger.RegelgruppeForRegistrerteOpplysninger
+
+class Hovedregler(personfakta: Personfakta) {
+
+    private val regelgruppeForRegistrerteOpplysninger = RegelgruppeForRegistrerteOpplysninger(personfakta)
+    private val regelgruppeForGrunnforordningen = RegelgruppeForGrunnforordningen(personfakta)
+    private var reglerForArbeidsforhold = RegelgruppeForArbeidsforhold(personfakta)
+
+    fun evaluer() =
+            Spørsmål(
+                    identifikator = "LOVME",
+                    spørsmål = "Er brukeren omfattet av norsk lovvalg og medlem av folketrygden?",
+                    beskrivelse = "",
+                    operasjon = { kjørRegler() }
+            ).utfør()
+
+
+    private fun kjørRegler(): Svar {
+        val oppl = regelgruppeForRegistrerteOpplysninger.evaluer()
+
+        if (oppl.svar.resultat == Resultattype.JA) {
+            return Svar(
+                    resultat = Resultattype.UAVKLART,
+                    begrunnelse = "Bruker har registrerte opplysninger i NAV som må kontrolleres nærmere",
+                    underspørsmål = listOf(oppl)
+            )
+        }
+
+        val grunn = regelgruppeForGrunnforordningen.evaluer()
```

Comment:
Her ligger det en implisitt `if (oppl.svar.resultat == Resultattype.NEI) {`, hvorfor har du valgt å gjøre den implisitt i stedet for å beholde den eksplisitt slik det var før? Mindre nesting? Jeg er litt usikker på om jeg synes det har blitt mer lesbart..

Reply:
er nok det. skal prøve å gjøre det litt mer eksplisitt